src/js/components/InfiniteScroll/stories/Grid.js

```diff
@@ -13,7 +13,10 @@ export const GridInfiniteScroll = () => (
     <InfiniteScroll items={allItems} step={12}>
       {(item) => (
         <Box key={item} as="article" pad="xsmall">
-          <Image src="https://via.placeholder.com/350x150" />
+          <Image
+            src="https://via.placeholder.com/350x150"
+            alt="Placeholder Text"
```

Comment:
```suggestion
            alt="Placeholder Image"
```
Can we change this to say Image instead of Text?

Reply:
Yeah sorry about that, i was working on the TextInput as well and got confused.